tools.go

```diff
@@ -14,7 +14,7 @@
 
 //go:build tools
 
-package main
+package pinecone
```

Comment:
`paramgen` was updated so that `main` is allowed here: https://github.com/ConduitIO/conduit-connector-sdk/pull/120.